src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/ConfirmOutcome.cshtml

```diff
@@ -0,0 +1,216 @@
+@using SFA.DAS.AssessorService.Domain.Extensions;
+@using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+
+@model SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway.RoatpGatewayApplicationViewModel
+
+@{
+    ViewBag.Title = "Gateway - Overall gateway outcome";
+    Layout = "_Layout";
+}
+
+@{
+    var formGroupError = Model.ErrorMessages != null && Model.ErrorMessages.Any() ? "govuk-form-group--error" : "";
+    var radioCheckedAskClarification = Model.GatewayReviewStatus == "Ask Clarification" ? "checked" : "";
+    var radioCheckedDeclined = Model.GatewayReviewStatus == "Declined" ? "checked" : "";
+    var radioCheckedApproved = Model.GatewayReviewStatus == "Approved" ? "checked" : "";
+
+    var isGatewayReviewStatus = false;
+    var isErrorAskClarification = false;
+    var isErrorDeclined = false;
+    var isErrorApproved = false;
+
+    var isErrorAskClarificationCss = "";
+    var isErrorDeclinedCss = "";
+    var isErrorApprovedCss = "";
+
+    var ErrorGatewayReviewStatus = "";
+    var ErrorAskClarificationText = "";
+    var ErrorDeclinedText = "";
+    var ErrorApprovedText = "";
+}
+
+<div class="govuk-breadcrumbs">
+    <ol class="govuk-breadcrumbs__list">
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="/Dashboard">RoATP dashboard</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="/Roatp/Gateway/New">RoATP applications</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="/Roatp/Gateway/@Model.ApplicationId">Application assessment overview</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item" aria-current="page">Gateway outcome</li>
+    </ol>
+</div>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            @if (Model.ErrorMessages != null && Model.ErrorMessages.Any())
+            {
+                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
+                    <h2 class="govuk-error-summary__title" id="error-summary-title">
+                        There is a problem
+                    </h2>
+                    <div class="govuk-error-summary__body">
+                        <ul class="govuk-list govuk-error-summary__list">
+                            @foreach (var error in Model.ErrorMessages)
```

Comment:
I'm sure there's a simpler way of doing this by tagging the fields. Maybe one for Greg

Reply:
View is refactored. The ViewModelOnError is proccessed in GatewayOverviewOrchestrator.ProcessViewModelOnError()